access/access.go

```diff
@@ -211,16 +220,16 @@ func (c *clt) GetRequests(ctx context.Context, fltr Filter) ([]Request, error) {
 }
 
 func (c *clt) CreateRequest(ctx context.Context, user string, roles ...string) (Request, error) {
-	req := &services.AccessRequestV3{
-		Kind:    services.KindAccessRequest,
-		Version: services.V3,
-		Metadata: services.Metadata{
+	req := &types.AccessRequestV3{
```

Comment:
Use `services.NewAccessRequest(user, roles...)`, which is functionally equivalent.